### DISCUSSION TITLE
Lesson 1 - Deployed but not showing colours

### DISCUSSION THREAD

**Discussion by cryptofruits** [smartcontractkit/full-blockchain-solidity-course-py#1454] [Q&A] [answered]
Hello - I managed to deploy the contract but none of the script was in different colours.

```Solidity
// SPDX-License-Identifier: MIT

pragma solidity ^0.8.0;

contract SimpleStorage {

    //this will get intialized to 0!
    uint256 public favoriteNumber;
    
    function store(uint256 _favoriteNumber) public {
        favoriteNumber = _favoriteNumber;
    }

}

```
ALSO there was no STORE option after it was deployed - TY

**Comment by cromewar** [comment, marked as answer]
Hello @cryptofruits 
If you are deploying using remix, try deleting the cache of the browser and also trying with another one like firefox.
remember you have to compile the code each time with Ctrl + S on Remix, and the re deploy de contract with the changes made.

**Reply by cryptofruits** [reply]
Hi @cromewar thank you for your reply - I tried on another browser and got the following:

pragma solidity >=0.8.0 <0.9.0;

contract SimpleStorage {

    // this wil get intialized to 0!
    uint256 public favoriteNumber;

    function store(uint256 _favoriteNumber) public {
       favoriteNumber =  _favoriteNumber;  
Ther wer deinatly some change in the coluring BUT I still dont get the additional the STORE and FAVORITE NUMBER options ?  
Can you see what I am doing wrong
Thank you

**Reply by cryptofruits** [reply]
Found it whoop whoop ...TY

**Reply by cromewar** [reply]
I'm glad is solved now!.
